server/lib/util.py

```diff
@@ -253,6 +253,11 @@
     'chart_titles_by_sv.json', 'chart_titles_by_sv_sdg.json'
 ]
 
+# Filter out observations with dates in the future for these variable DCIDs
+# when finding the date of highest coverage
+FILTER_FUTURE_OBSERVATIONS_FROM_HIGHEST_COVERAGE_FOR_VARIABLE_DCIDS = set(
```

Comment:
can we do a slightly shorter name?

Reply:
changed to `FILTER_FUTURE_OBSERVATIONS_FROM_VARIABLES`